workflows_core/engine/stable_engine.py

```diff
@@ -85,18 +93,23 @@ def apply(self) -> None:
                     successful_chunks += 1
                     chunk_to_update.extend(new_batch)
 
-            # We want to make sure the schema updates
-            # on the first chunk upserting
-            if chunk_counter < self.MAX_SCHEMA_UPDATE_LIMITER:
-                ingest_in_background = False
+            if self.output_to_status:
+                # Store in output documents
+                self.extend_output_documents(document.to_json() for document in chunk_to_update)
```

Comment:
`self.extend_output_documents([document.to_json() for document in chunk_to_update])`